### PR TITLE
mds: remove the misleading and redundant logs

### PR DESCRIPTION
**Pull request by lxbsz**
## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: src/mds/Server.cc [hunk 1]
```diff
@@ -4214,8 +4177,49 @@ void Server::handle_client_getattr(const MDRequestRef& mdr, bool is_lookup)
     }
   }
 
-  if (!mds->locker->acquire_locks(mdr, lov))
+  if (!mds->locker->acquire_locks(mdr, lov)) {
```
Comment (vshankar) [inline src/mds/Server.cc:4180]: Could you explain this change @lxbsz? Seems like this is more than just removing misleading logs?

Comment (lxbsz) [reply src/mds/Server.cc:4180]: Oh, sorry for late. This just a mistake. I will remove it.

File: src/mds/MDCache.cc [hunk 2]
```diff
@@ -8520,7 +8520,6 @@ int MDCache::path_traverse(const MDRequestRef& mdr, MDSContextFactory& cf,
 	  lov.add_rdlock(&dn->lock);
 	}
 	if (!mds->locker->acquire_locks(mdr, lov)) {
-	  dout(10) << "traverse: failed to rdlock " << dn->lock << " " << *dn << dendl;
 	  return 1;
```
Comment (dparmar18) [inline src/mds/MDCache.cc:8523]: I understand that this is beyond the scope of this PR but are we sure we want to return `1`? This should be `CEPHFS_EPERM`

Comment (batrick) [reply src/mds/MDCache.cc:8523]: errors are negative, so no?

**Review by batrick — APPROVED** [review]

**Comment by batrick** [comment]
jenkins test api

**Comment by batrick** [comment]
jenkins test api

**Comment by batrick** [comment]
This PR is under test in [https://tracker.ceph.com/issues/67214](https://tracker.ceph.com/issues/67214).

**Pull request merged**